src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx

```diff
@@ -15,14 +15,26 @@ export default class GuideDrawer extends React.Component {
     step: PropTypes.number.isRequired,
     onFinish: PropTypes.func.isRequired,
     onDismiss: PropTypes.func.isRequired,
+    orgSlug: PropTypes.string,
   };
 
   handleUseful = useful => {
     markUseful(this.props.guide.id, useful);
     this.props.onFinish();
   };
 
+  interpolate(template, variables) {
+    let regex = /\${([^{]+)}/g;
+    return template.replace(regex, (match, g1) => {
+      return variables[g1.trim()];
+    });
+  }
+
   render() {
+    let message_variables = {
+      org_slug: this.props.orgSlug,
```

Comment:
we generally do `messageVariables` and `orgSlug` because javascript

Reply:
Sure, fixed.